webapp/src/ts/modals/edit-user/confirm-password-updated.component.html

```diff
@@ -0,0 +1,8 @@
+<mm-modal
+  [titleKey]="'Confirm'"
+  [submitKey]="'Ok'"
+  [cancelKey]="false"
+  (onSubmit)="submit()"
```

Comment:
You're missing the `onCancel` output here. Without it, the little X in the top right corner doesn't do anything. 

![image](https://user-images.githubusercontent.com/35681649/106469661-40d64680-64a8-11eb-9be4-d72d11f80155.png)

Reply:
That is my fault :facepalm: , I have [suggested to remove](https://github.com/medic/cht-core/pull/6841/files/9954c2b5432fcfd1712a2d4b1ff61f1b748e7f6e#diff-8b66b1f63316931d52add92e47229fb1bc0d312c1f7fd0f30c6643e32f499184) the "cancel" button setting it to `false` (which is OK) along with the removal of the `onCancel` attribute (wrong).

Reply:
Done, thanks.